packages/web-components/src/tree-item/tree-item.ts

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import { attr, customElement, FASTElement, observable, volatile } from '@microsoft/fast-element';
+import { FluentDesignSystem } from '../fluent-design-system.js';
+import { template } from './tree-item.template.js';
+import { styles } from './tree-item.style.js';
+
+@customElement({
+  name: `${FluentDesignSystem.prefix}-tree-item`,
+  template,
+  styles,
+})
```

Comment:
The current implementation of registration doesn't leverage the decorator pattern. I see that you have definitions included, this can likely be removed in favor of those definitions.

Reply:
What's your prefer name?